### PR TITLE
Fix participants age for tokyoSkyra site

### PR DESCRIPTION
**Pull request by alexfoias**
Updated the information based on the information from our collaborators.

I tried to trace back when did the issue appear, but it was impossible due to the various changes in infrastructure.

### CODE REVIEW

**Comment by jcohenadad** [comment]
> I tried to trace back when did the issue appear, but it was impossible due to the various changes in infrastructure.

you need to trace it back because we need to make sure:
- the participant IDs did not change (ie: the info given by Kouhei recently are valid for this BIDS dataset)
- this error did not happen for other datasets

so please, go back in time, look for emails, etc. document as much as possible. This is a very serious issue and we have a responsibility to not disseminate fake results.

**Comment by jcohenadad** [comment]
> * this error did not happen for other datasets

e.g.: the tokyoIngenia values are also suspicious:

~~~
sub-tokyoIngenia01	M	25	-	-	2019-10-01	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	-	5.3.1_5.3.1.1	"K. Kamiya, Y. Suzuki"
sub-tokyoIngenia02	F	32	-	-	2019-10-01	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	-	5.3.1_5.3.1.1	"K. Kamiya, Y. Suzuki"
sub-tokyoIngenia03	M	36	-	-	2019-10-01	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	-	5.3.1_5.3.1.1	"K. Kamiya, Y. Suzuki"
sub-tokyoIngenia04	F	29	-	-	2019-10-01	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	-	5.3.1_5.3.1.1	"K. Kamiya, Y. Suzuki"
sub-tokyoIngenia05	F	28	-	-	2019-10-01	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	-	5.3.1_5.3.1.1	"K. Kamiya, Y. Suzuki"
sub-tokyoIngenia06	M	24	-	-	2019-06-13	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	-	5.3.1_5.3.1.1	"K. Kamiya, Y. Suzuki"
sub-tokyoIngenia07	M	35	-	-	2019-10-01	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	-	5.3.1_5.3.1.1	"K. Kamiya, Y. Suzuki"
~~~

**Comment by alexfoias** [comment]
I checked in all my emails. I could only find demographics for Juntendo. 

I think we received the data already converted to nifti.  In the email conversation with @jcohenadad with the subject `Re: Spinal cord generic protocol : consent from participants @ Tokyo Univ` sent on `Wed, Apr 3, 2019 at 1:40 PM`, I asked if we had the raw data, but I didn't get an answer. 

In the changes files, we don't have all the history of changes (the one from openeuro).

I don't know who added the data to the dataset.

**Comment by alexfoias** [comment]
This error seemed to be in the initial commit on openneuro https://github.com/OpenNeuroDatasets/ds002902

**Comment by jcohenadad** [comment]
> I checked in all my emails. I could only find demographics for Juntendo.
> 
> I think we received the data already converted to nifti. In the email conversation with @jcohenadad with the subject `Re: Spinal cord generic protocol : consent from participants @ Tokyo Univ` sent on `Wed, Apr 3, 2019 at 1:40 PM`, I asked if we had the raw data, but I didn't get an answer.

sorry for the oversight-- the answer is: no, we don't have the raw data

> In the changes files, we don't have all the history of changes (the one from openeuro).
> 
> I don't know who added the data to the dataset.

me neither

**Comment by jcohenadad** [comment]
image

OK, we know these are the same subjects. So the last sanity check is to verify that the sub01 on tokyoIngenia corresponds to sub01 on tokyoSkyra, etc.  @alexfoias can you please do that (look at the T1w MRIs and you will easily find out if these are the same subjects)

**Comment by alexfoias** [comment]
I did a visual checkout and the subjects seem to be correctly identified (similar anatomy for all 3 centers).

**Review by jcohenadad — APPROVED** [review]

**Review by kousu — CHANGES_REQUESTED** [review]
Because 

> we have a responsibility to not disseminate fake results.

this entire discussion needs to be summarized into the commit message before this is merged. Maybe even expanded, not summarized: include email addresses, institution names, URLs, dates, explanations, everything to explain where the error came from, how it was resolved.

That's the big benefit of using git diligently -- so that future us *can* trace back what happened to a dataset, and why.

**Comment by kousu** [comment]
Hey, I fixed CI, so now we can actually talk about reviewing this properly.

1. What is "the information from our collaborators" that triggered this update? Who are they? Names and academic email addresses?
2. The `tokyoIngenia` values also need to be synchronized.
3. Should we delete https://openneuro.org/datasets/ds002902? Since it's wrong?
    a. If we do delete it, how do we make sure it's also deleted from https://github.com/OpenNeuroDatasets/ds002902/? I know their Github syncer is pretty flakey sometimes.
4. Let's make sure to write a relatively _detailed_ commit message justifying this change.


Also, this is probably worth a different issue, but:

5. Should we rearrange the dataset to put all the data from the same physical people under the same (e.g. `sub-tokyo*01` -> `sub-tokyo01`, `sub-vuiis*04` -> `sub-vuiss04`) and represent the scanner used as a tag in the filenames?

**Comment by jcohenadad** [comment]
> Should we delete https://openneuro.org/datasets/ds002902? Since it's wrong?

yes ( i thought we already did it @alexfoias ?)

> Should we rearrange the dataset to put all the data from the same physical people under the same

sorry i don't understand your suggestion-- what do you mean by "data from the same physical people"?

**Comment by alexfoias** [comment]
I have removed all datasets from OpenNeuro. (This screenshot is take from the generic account used to create the datasets.)
Screen Shot 2021-11-15 at 11 00 33 AM
.

**Comment by kousu** [comment]
> I have removed all datasets from OpenNeuro.

:facepalm:  you're right. Okay, punting: https://github.com/OpenNeuroOrg/openneuro/issues/2484


> (This screenshot is take from the generic account used to create the datasets.)

By the way does anyone else have that credential? Does Julien? Should I? Better yet, can we make a Team on openneuro and all have our own credentials instead? Or are we giving up on OpenNeuro entirely?

**Comment by kousu** [comment]
> > Should we delete https://openneuro.org/datasets/ds002902? Since it's wrong?
> 
> yes ( i thought we already did it @alexfoias ?)
> 

The older version is off OpenNeuro for good now: https://github.com/OpenNeuroOrg/openneuro/issues/2484


> > Should we rearrange the dataset to put all the data from the same physical people under the same
> 
> sorry i don't understand your suggestion-- what do you mean by "data from the same physical people"?

-> https://github.com/spine-generic/data-multi-subject/issues/102

>     2. The `tokyoIngenia` values also need to be synchronized.

And all `tokyo750w*` too!: 10c5e459ffb494b10e8054191cffeb6e20e281e8

Still **TODO**: writing a good commit message.

>   1. What is "the information from our collaborators" that triggered this update? Who are they? Names and academic email addresses?
>   4. Let's make sure to write a relatively _detailed_ commit message justifying this change.

@alexfoias could we see the email that inspired this PR?

**Comment by kousu** [comment]
New CI bug? https://github.com/spine-generic/data-multi-subject/runs/4226501836?check_suite_focus=true#step:11:2598

> ```
>  (recording state in git...)
> git-annex: get: 3 failed
> Error: Process completed with exit code 1.
> ```

Looks like maybe it's Amazon S3 having a minor outage?

https://github.com/spine-generic/data-multi-subject/runs/4226501836?check_suite_focus=true#step:11:620

```
get derivatives/labels/sub-stanford04/anat/sub-stanford04_acq-T1w_MTS_seg-manual.nii.gz (from amazon...) 
  download failed: Internal Server Error

  Unable to access these remotes: amazon

  Try making some of these repositories available:
  	5a5447a8-a9b8-49bc-8276-01a62632b502 -- [amazon]
   	5cdba4fc-8d50-4e89-bb0c-a3a4f9449666 -- julien@julien-macbook.local:~/code/spine-generic/data-multi-subject
   	9e4d13f3-30e1-4a29-8b86-670879928606 -- alex@NeuroPoly-MacBook-Pro.local:~/data/data-multi-subject

  (Note that these git remotes have annex-ignore set: origin)
failed
```

https://github.com/spine-generic/data-multi-subject/runs/4226501836?check_suite_focus=true#step:11:658

```
get derivatives/labels/sub-tehranS01/anat/sub-tehranS01_T1w_labels-disc-manual.nii.gz (from amazon...) 
  download failed: Internal Server Error

  Unable to access these remotes: amazon

  Try making some of these repositories available:
  	5a5447a8-a9b8-49bc-8276-01a62632b502 -- [amazon]
   	5cdba4fc-8d50-4e89-bb0c-a3a4f9449666 -- julien@julien-macbook.local:~/code/spine-generic/data-multi-subject
   	9e4d13f3-30e1-4a29-8b86-670879928606 -- alex@NeuroPoly-MacBook-Pro.local:~/data/data-multi-subject

  (Note that these git remotes have annex-ignore set: origin)
failed
```

https://github.com/spine-generic/data-multi-subject/runs/4226501836?check_suite_focus=true#step:11:711

```
get derivatives/labels/sub-tokyoSkyra03/anat/sub-tokyoSkyra03_T2star_rms_gmseg-manual.nii.gz (from amazon...) 
  download failed: Internal Server Error

  Unable to access these remotes: amazon

  Try making some of these repositories available:
  	5a5447a8-a9b8-49bc-8276-01a62632b502 -- [amazon]
   	5cdba4fc-8d50-4e89-bb0c-a3a4f9449666 -- julien@julien-macbook.local:~/code/spine-generic/data-multi-subject
   	9e4d13f3-30e1-4a29-8b86-670879928606 -- alex@NeuroPoly-MacBook-Pro.local:~/data/data-multi-subject

  (Note that these git remotes have annex-ignore set: origin)
failed
```


The following run passed: https://github.com/spine-generic/data-multi-subject/runs/4226604983?check_suite_focus=true

**Comment by alexfoias** [comment]
@kousu Here is the info from the email:

Subject01 M 36 177cm 70kg
Subject02 M 36 168cm 80kg
Subject03 M 32 162cm 57kg
Subject04 F 34 161cm 65kg
Subject05 F 35 154cm 51kg
Subject06 F 30 160cm 48kg
Subject07 M 29 165cm 60kg

**Comment by jcohenadad** [comment]
@alexfoias can you please update this PR after https://github.com/spine-generic/data-multi-subject/pull/112

**Review by alexfoias — COMMENTED** [review]
The values are the ones transferred by the collaborator.

File: participants.tsv [hunk 1]
```diff
@@ -200,27 +200,28 @@ sub-tehranS03	F	30	170	65	2019-02-27	tehran	National_Brain_Mappin_Lab_Iran	Sieme
...
-sub-tokyoIngenia07	M	35	165	60	2019-10-01	tokyo	Ingenia the University of Tokyo	Philips	Ingenia	n/a	5.3.1_5.3.1.1	K. Kamiya, Y. Suzuki
-sub-tokyoSkyra01	M	25	177	70	2019-10-01	tokyoSkyra	the University of Tokyo	Siemens	Skyra	HeadNeck_20	syngo_MR_E11	K. Kamiya, Y. Suzuki
-sub-tokyoSkyra02	F	32	168	80	2019-04-05	tokyoSkyra	the University of Tokyo	Siemens	Skyra	HeadNeck_20	syngo_MR_E11	K. Kamiya, Y. Suzuki
-sub-tokyoSkyra03	M	36	162	57	2019-10-01	tokyoSkyra	the University of Tokyo	Siemens	Skyra	HeadNeck_20	syngo_MR_E11	K. Kamiya, Y. Suzuki
-sub-tokyoSkyra04	F	29	161	65	2019-10-01	tokyoSkyra	the University of Tokyo	Siemens	Skyra	HeadNeck_20	syngo_MR_E11	K. Kamiya, Y. Suzuki
-sub-tokyoSkyra05	F	28	154	51	2019-10-01	tokyoSkyra	the University of Tokyo	Siemens	Skyra	HeadNeck_20	syngo_MR_E11	K. Kamiya, Y. Suzuki
-sub-tokyoSkyra06	M	24	160	48	2019-10-01	tokyoSkyra	the University of Tokyo	Siemens	Skyra	HeadNeck_20	syngo_MR_E11	K. Kamiya, Y. Suzuki
-sub-tokyoSkyra07	M	35	165	60	2019-06-13	tokyoSkyra	the University of Tokyo	Siemens	Skyra	HeadNeck_20	syngo_MR_E11	K. Kamiya, Y. Suzuki
+
```
Comment (jcohenadad) [inline participants.tsv]:
```suggestion
```

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**